src/views/CanHelp/Outlet.vue

```diff
@@ -75,6 +67,12 @@
                                         Materiały do szycia:
                                         <strong>tak</strong>
                                     </div>
+
+                                    <div>
+                                        Transport:
+                                        <strong>tak</strong>
+                                    </div>
```

Comment:
Is transport always there ? shouldn't it be 
```
<div v-if="item.transport">
  Transport: 
  <strong>tak</strong>
</div>
```

Reply:
or maybe it's because backend is not yet ready, and is not returning it ?

Reply:
You're right, I've missed it.